Jenkinsfile

```
@@ -15,7 +15,7 @@ pipeline {
             role: worker
           containers:
           - name: ecs-pipeline
-            image: quay.io/uktrade/ecs-pipeline
+            image: 165562107270.dkr.ecr.eu-west-2.amazonaws.com/ecs-pipeline
```

Should these be configurable via pipeline parameters maybe?

e.g.

```suggestion
            image: ${env.AwsAccount}.dkr.ecr.${env.AwsRegion}.amazonaws.com/ecs-pipeline
```

I'm not _totally_ sure that would work, but I think it should?